.github/workflows/deploy-book.yml

```
@@ -0,0 +1,58 @@
+# https://jupyterbook.org/en/stable/publish/gh-pages.html#automatically-host-your-book-with-github-actions
+name: deploy-book
+
+# Only run this when the main branch changes
+on:
+  # Uncomment the 'pull_request' line below to manually re-build Jupyter Book
+  pull_request:
```

Need to comment this line before merging into main branch.
```suggestion
  # pull_request:
```